### PR TITLE
Week4's files added.

### PR DESCRIPTION
**Pull request by LucasJSch**
The description of the 3D model in the '.md' file made is still missing. Same with the 'ROS REP' section.

### CODE REVIEW

File: lucas_scheinkerman/Robot_modeling.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+### urdf_to_graphiz
+
+This is another tool which allows to visualize in an easy way the links relations, similar to the previous one but it creates a `.pdf` file which has the tree structure converted into image. The same tree that was shown before, now is showed as the output of the `urdf_to_graphiz` command:
+
+![urdf_to_graphiz_output](media/urdf_to_graphiz.png)
+
+It can be seen that additionally to the tree relation, the output shows the `xyz` and `rpy` vectors that corresponds to every joint in relation to its parent link.
+
+## Simulating links as light sensors
```
Comment (eborghi10) [inline lucas_scheinkerman/Robot_modeling.md:75]: Avoid pushing code here. It's fine if you put a link to the PR that adds the code.

**Review by eborghi10 — CHANGES_REQUESTED** [review]

**Comment by LucasJSch** [comment]
# Ready for review

File: lucas_scheinkerman/Robot_modeling.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
+# Robot modeling
+
+In this week we were assigned to investigate different subject regarding robot moldeing in ROS and Gazebo, and as a final assignment for this week we were asked to model two identical links that simulate light sensors.
```
Comment (eborghi10) [inline lucas_scheinkerman/Robot_modeling.md:3]: Typo: modeling.

**Review by eborghi10 — APPROVED** [review]
One minor comment, but LGTM.

**Pull request merged**